spec/vehicle_locations.schema.json

```diff
@@ -38,12 +38,21 @@
       "description": "Identifies the trip performed."
     },
     {
-      "name": "stop_sequence",
+      "name": "trip_stop_sequence",
+      "type": "integer",
+      "description": "The temporal sequence of the stop visit that the vehicle is serving or approaching within its performed trip, starting at 1 and containing no gaps. This may differ from scheduled_stop_sequence.",
+      "constraints": {
+        "required": true,
```

Comment:
I see we're transferring the required constraint to `trip_stop_sequence`. However, if `trip_id_performed` isn't required I'm not sure how to make sense of `trip_stop_sequence` being required. Let's drop the required constraint on this field.